.gitignore

```diff
@@ -3,7 +3,8 @@ _build
 **/CMakeFiles
 **/CMakeCache.txt
 *.pyc
-sdk/
+**/build
+/sdk/
```

Comment:
I'm not sure how git behaves, but is the absolute path here relative to the git repo?

Reply:
This is for ignoring the `sdk` folder in root but not ignoring the `sdk` folder in src/sdk @bencefr

Reply:
From the .ignore doc: *A leading slash matches the beginning of the pathname. For example, "/*.c" matches "cat-file.c" but not "mozilla-sha1/sha1.c".*